src/com/dotmarketing/portlets/contentlet/business/web/ContentletWebAPIImpl.java

```diff
@@ -183,12 +185,19 @@ public String saveContent(Map<String, Object> contentletFormData,
 		if(autocommit)
 		    HibernateUtil.commitTransaction();
 
+		ContentletSystemEventUtil.getInstance().pushSaveEvent(user, cont, isNew);

```

Comment:
this should be an attribute class, instead of using a static reference, please assign it to a private final ContentletSystemEventUtil, so it will be easier to mock